### ISSUE TITLE
Deer Antlers ought to also unlock Gore Attacks

### ISSUE THREAD

**Issue by Drake713** [Kitteh6660/Corruption-of-Champions-Mod#827]
### Game version
	e.g.: 1.0.2_mod_1.4.6b

I couldn't find any info on if they did or did not, so I turned on debug mode, used a goldenrind till my antlers had 30 points, then did a test battle and the Gore special attack was not available.

**Comment by Drake713** [comment]
Also seems like the gore attack could use some code clean up; enemy damage resistance is applied 2x in the damage calculation

**Comment by CapriciousClowns** [comment]
yeah it be cool if you could like head butt or just pound your horns match to match with another horned creature like a minotaur show off dat sweet sexy spiky horns also when i read it in the character description it always reminds me of aku horns
